packages/venia-concept/src/components/Checkout/form.js

```diff
@@ -1,22 +1,22 @@
 import React, { Component, Fragment } from 'react';
 import { array, bool, func, object, shape, string } from 'prop-types';
 
-import { Price, Util } from '@magento/peregrine';
+import { Price } from '@magento/peregrine';
 import AddressForm from './addressForm';
 import PaymentsForm from './paymentsForm';
 import Section from './section';
 import ShippingForm from './shippingForm';
 import SubmitButton from './submitButton';
 
 import classify from 'src/classify';
+import Button from 'src/components/Button';
 import defaultClasses from './form.css';
 
-const { BrowserPersistence } = Util;
-const storage = new BrowserPersistence();
```

Comment:
It's always great to see one of these going away. 🙌

Reply:
Yeah I moved this logic up to the wrapping component `Checkout/wrapper.js`, which is now the only one hooked up to Redux and aware of storage. It passes all that information down to the checkout form via `props`.